### PR TITLE
OSDOCS#8971: Adding release note for secret generation behavior change

### PR DESCRIPTION
**Pull request by bergerhoffer**
Version(s):
4.15

Issue:
https://issues.redhat.com/browse/OSDOCS-8971

Link to docs preview:
https://71257--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-15-release-notes#ocp-4-15-auth-generated-secrets

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-15-release-notes.adoc
Comment (bergerhoffer) [inline release_notes/ocp-4-15-release-notes.adoc:601]: Note to reviewers: the link text doesn't match the page currently, but it will match once #71256 is merged.

**Review by sanchezl — APPROVED** [review]

**Comment by gangwgr** [comment]
seems ok to me

**Comment by smunje1** [comment]
Note: Before merge review request, we need to resolve [this comment](https://github.com/openshift/openshift-docs/pull/71257#discussion_r1483573365). Thank you!

@bergerhoffer This changes LGTM. As I am new to the peer review squad, I request that you wait until @bscott-rh gives approval.

**Comment by bscott-rh** [comment]
> Note: Before merge review request, we need to resolve [this comment](https://github.com/openshift/openshift-docs/pull/71257#discussion_r1483573365). Thank you!
> 
> @bergerhoffer This changes LGTM. As I am new to the peer review squad, I request that you wait until @bscott-rh gives approval.

LGTM.

**Pull request merged**
